docs/Tutorials/Examples/Privacy-Example.md

```diff
@@ -1,21 +1,21 @@
-description: Hyperledger Besu private network with privacy enabled quickstart tutorial
+description: Hyperledger Besu private network with privacy enabled tutorial
 <!--- END of page meta data -->
 
-# Private Network with Privacy Enabled Quickstart Tutorial
+# Private Network with Privacy Enabled Example Tutorial
```

Comment:
I know you are just following from what was there before but should be "Privacy-enabled Example Tutorial"

Reply:
This would be opposed to a "Public network with privacy enabled" as we get closer to having our privacy solutions be mainnet compatible